src/sidebar/components/test/group-list-item-test.js

```diff
@@ -43,55 +59,58 @@ describe('GroupListItem', () => {
     return mount(
       <GroupListItem
         group={fakeGroup}
+        groups={fakeGroups}
         analytics={fakeAnalytics}
-        store={fakeStore}
       />
     );
   };
 
   it('changes the focused group when group is clicked', () => {
-    const fakeGroup = { id: 'groupid' };
-
     const wrapper = createGroupListItem(fakeGroup);
-    wrapper.find('.group-list-item__item').simulate('click');
+    wrapper
```

Comment:
You could do this with `shallow` rendering, too, no?

Reply:
I ran into an issue with that. IIRC it used to be the case that `GroupListItem` was a leaf component, which is why it used `mount` originally. Trying to switch it to `shallow` just now I ran into a limitation of Enzyme's shallow rendering to do with calling `wrapper.update()` [1] with a component that renders multiple elements at the top level.

[1] `wrapper.update()` synchronizes Enzyme's view of the rendered output from a component with the actual rendered output. This is called automatically when using eg. `wrapper.simulate(...)` but needs to be called manually if you do something which causes a re-render without Enzyme being aware of it.

Reply:
I've added a note about that issue